includes/class-pubsubhubbub-publisher.php

```diff
@@ -117,7 +117,13 @@ public static function publish_to_hub( $feed_urls ) {
 	 */
 	public static function get_hubs() {
 		$endpoints = get_option( 'pubsubhubbub_endpoints' );
-		$hub_urls  = explode( PHP_EOL, $endpoints );
+
+		// Fail with an empty array when option retrieval fails
+		if ( ! $endpoints ) {
+			$hub_urls = array();
```

Comment:
Shouldn't we use the default hubs in that case?

```suggestion
			$hub_urls = Pubsubhubbub::DEFAULT_HUBS;
```